### ISSUE TITLE
Microsoft.NET Framework

### ISSUE THREAD

**Issue by mtrivettpoker** [dlrudie/Snip#90]
Snip will not open and get following message every time I try to open:

Could not load file or assembly 'Accessibility, Version=4.0.0.0, Culture=neutral,PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.  The system cannot find the file specified.  

Was working fine until today.

**Comment by dlrudie** [comment]
Sounds like a change on your system and not Snip.
